### DISCUSSION TITLE
Convex

### DISCUSSION THREAD

**Discussion by aokigit** [coollabsio/coolify#3527] [Service Template Requests]
[Convex](https://www.convex.dev/) would be a great addition to Coolify!

**Comment by FelipeJz** [comment]
Right now convex-backend is not on Docker (https://github.com/get-convex/convex-backend/issues/6)
You could build an image and host it in a registry i guess, is just a binary

**Reply by IFTTD** [reply]
It is now. 
https://github.com/get-convex/convex-backend/tree/main/self-hosted
